boards/arm/ip_k66f/Kconfig.defconfig

```diff
@@ -0,0 +1,43 @@
+# IP-K66F board
+
+# Copyright (c) 2020 DENX Software Engineering GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_IP_K66F
+
+config BOARD
+	default "ip_k66f"
```

Comment:
Please intent by one tab instead of two spaces

Reply:
@MaureenHelm It was in tab, but just re-confirmed it. Thanks.